### DISCUSSION TITLE
client detected as virus

### DISCUSSION THREAD

**Discussion by EspectroW94** [AscensionGameDev/Intersect-Engine#2233] [General]
Some antivirus and browsers are detecting the client as a virus when downloading.

I leave the log here: https://www.virustotal.com/gui/file/721e1505a3b57aeba046a2321c5d809b7e6c8ebe3c97b0921a1ca4df84d54738/detection

**Comment by PyroTech03** [comment]
It's a false positive because the builds aren't signed. Just wait and hope the next build hash doesn't trigger, or build and sign yourself.
